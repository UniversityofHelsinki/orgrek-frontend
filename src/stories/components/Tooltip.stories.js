import React from 'react';
import TooltipComponent from '@mui/material/Tooltip';
import IconButton from '@mui/material/IconButton';
import DeleteIcon from '@mui/icons-material/Delete';

export default {
  component: TooltipComponent,
};

export const Tooltip = {
  args: {
    title: 'Delete',
  },
  render: (args) => (
    <TooltipComponent {...args}>
      <IconButton>
        <DeleteIcon />
      </IconButton>
    </TooltipComponent>
  ),
};
