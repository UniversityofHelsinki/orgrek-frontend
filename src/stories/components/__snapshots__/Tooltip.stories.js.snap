// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stories/components/Tooltip Tooltip smoke-test 1`] = `
Object {
  "checked": undefined,
  "children": Array [
    Object {
      "checked": undefined,
      "children": undefined,
      "name": "Delete",
      "pressed": undefined,
      "role": "button",
      "value": undefined,
    },
  ],
  "name": "stories-components-tooltip--tooltip",
  "pressed": undefined,
  "role": "WebArea",
  "value": undefined,
}
`;
